src/AddIns/Uno.UI.MSAL/ts/MSAL.WebUI.ts

```diff
@@ -0,0 +1,115 @@
+﻿namespace MSAL {
+	export class WebUI {
+
+		public static authenticate(urlNavigate: string, urlRedirect: string, title: string, popUpWidth: number, popUpHeight: number): Promise<string> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5669754)